### ISSUE TITLE
Can't use with mpp pen

### ISSUE THREAD

**Issue by VoreckLukas** [jjsullivan5196/wvkbd#57]
So I can type with my fingers like normal but If i want to use my mpp Pen (so i dont leave fingerprints on the screen of my convertible) the keyboard doesnt react

**Comment by proycon** [comment]
I'm not sure what causes this, wvkbd reacts to both touch and pointer events (which are separate things in wayland). Try to see what your pen outputs using a tool like [wev](https://git.sr.ht/~sircmpwn/wev) ?

**Comment by topaxi** [comment]
I tried using wvkbd with a pen too, it seems that `wev` doesn't output anything when using the pen. But using the pen to select text in an application, like Firefox (verified to run natively on wayland) is not an issue.

I assume pens on a tablet might be different from pointer and touch events, somehow 🤔 

wvkbd works fine with touch and mouse.

```
mice:
        Mouse at 565056635e80:
                elan06fa:00-04f3:317c-mouse
                        default speed: 0.00000
        Mouse at 5650565e10c0:
                elan06fa:00-04f3:317c-touchpad
                        default speed: 0.00000

Tablets:
        Tablet at 56505663b310:
                wacom-hid-52fb-pen
                        size: 286.04x178.78mm


Touch:
        Touch Device at 565056632030:
                wacom-hid-52fb-finger
```

**Comment by proycon** [comment]
@topaxi You're right, it looks like a tablet is different thing in wayland (just like touch and pointer are deliberately separated): https://wayland.app/protocols/tablet-v2 . Allowing virtual keys to be pressed with a pen on a tablet sounds like a valid use-case for wvkbd. I don't own any myself to test, which complicates development, but any patches are welcome!
